core/src/sql/value/serde/ser/statement/define/function.rs

```diff
@@ -93,6 +97,7 @@ impl serde::ser::SerializeStruct for SerializeDefineFunctionStatement {
 			comment: self.comment,
 			permissions: self.permissions,
 			if_not_exists: self.if_not_exists,
+			overwrite: self.overwrite,
```

Comment:
Not related to this PR, but thinking about serialisation, at some point, it would be nice to have the serialisation bing able to represents boolean in 1 bit (rather than 1 byte).